include/wx/qt/font.h

```diff
@@ -50,23 +50,29 @@ class WXDLLIMPEXP_CORE wxFont : public wxFontBase
                 wxFontEncoding encoding = wxFONTENCODING_DEFAULT);
 
     // accessors: get the font characteristics
+    virtual int GetPointSize() const wxOVERRIDE;
     virtual float GetFractionalPointSize() const wxOVERRIDE;
+    virtual wxSize GetPixelSize() const wxOVERRIDE;
     virtual wxFontStyle GetStyle() const;
     virtual int GetNumericWeight() const wxOVERRIDE;
     virtual bool GetUnderlined() const;
     virtual wxString GetFaceName() const;
     virtual wxFontEncoding GetEncoding() const;
     virtual const wxNativeFontInfo *GetNativeFontInfo() const;
+    virtual bool GetStrikethrough() const wxOVERRIDE;
 
     // change the font characteristics
     virtual void SetFractionalPointSize(float pointSize) wxOVERRIDE;
+    virtual void SetPixelSize(const wxSize& pixelSize) wxOVERRIDE;
     virtual void SetFamily( wxFontFamily family );
     virtual void SetStyle( wxFontStyle style );
     virtual void SetNumericWeight(int weight) wxOVERRIDE;
     virtual bool SetFaceName(const wxString& facename);
     virtual void SetUnderlined( bool underlined );
+    virtual void SetStrikethrough(bool strikethrough) wxOVERRIDE;
     virtual void SetEncoding(wxFontEncoding encoding);
-    
+    bool SetNativeFontInfo(const wxString& info);
```

Comment:
This doesn't seem right, we shouldn't redefine a non-virtual base class function in a derived class.

Reply:
Correct, I will remove this!